src/Injector.ts

```diff
@@ -44,6 +53,17 @@ export function Injector<C, T extends Constructor<BaseInjector<C>>>(Base: T, con
 			super(context);
 		}
 
+		@afterRender()
+		protected decoratateBind(node: DNode) {
+			const { bind } = this.properties;
+			decorate(node, (node: HNode | WNode<DefaultWidgetBaseInterface>) => {
+				const { properties = {} }: { properties: { bind?: any } } = node;
```

Comment:
Me and my annoying branch coverage... It is likely we aren't testing where bind is actually present, though if we are relying on typings it shouldn't be there.  We might have to instanbul ignore something here.